src/components/Messages/Message/Message.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import "./Message.scss";
+import { MessageData } from 'solarpunk-swarm-decentralized-chat';
+import AvatarMonogram from '../../AvatarMonogram/AvatarMonogram';
+import { useGlobalState } from '../../../GlobalStateContext';
+import LikeIcon from '../../icons/LikeIcon/LikeIcon';
+import { createMonogram, formatTime } from '../../../utils/helpers';
+
+interface MessageProps {
+    data: MessageData;
+    showTime?: boolean;
+    threadId: string;
+}
+
+const Message: React.FC<MessageProps> = ({
+    data,
+    showTime,
+    threadId,
+}) => {
+  const mockMonogram = "OP";
```

Comment:
Remove this

Reply:
threadId will be needed, others I will delete, thread logic is not implemented yet.

Reply:
removed showTime and mockMonogram